docs/upgrade.html

```diff
@@ -43,6 +43,10 @@ <h5><a id="upgrade_360_notable" href="#upgrade_360_notable">Notable changes in 3
             See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-925%3A+Rack+aware+task+assignment+in+Kafka+Streams">KIP-925</a>
             and <a href="/{{version}}/documentation/streams/developer-guide/config-streams.html#rack-aware-assignment-strategy"><b>Kafka Streams Developer Guide</b></a> for more details.
         </li>
+        <li>To account for a break in compatibility introduced in version 3.1.0, MirrorMaker 2 has added a new
+            <a href="#mirror_connector_replication.policy.internal.topic.separator.enabled">replication.policy.internal.topic.separator.enabled</a>
+            property. If upgrading from 3.0.x or earlier, it may be necessary to set this property to <code>false</code>; see the property's
+            <a href="#mirror_connector_replication.policy.internal.topic.separator.enabled">documentation</a> for more details.</li>
```

Comment:
We can also link to [KIP-949](https://cwiki.apache.org/confluence/display/KAFKA/KIP-949%3A+Add+flag+to+enable+the+usage+of+topic+separator+in+MM2+DefaultReplicationPolicy) here, but I'm hoping that the information provided here and in the property's docstring should be sufficient to save people the time and effort of reading through an entire KIP instead.

Reply:
It does look like the other items in this section include KIP links, but in general there aren't too many KIP links in the upgrade section. I think linking the KIP is unnecessary.